src/derivkit/forecasting/priors/core.py

```diff
@@ -0,0 +1,851 @@
+"""Prior utilities (core priors + unified builder).
+
+Priors are represented as callables:
+
+    ``logprior(theta) -> float``
```

Comment:
```suggestion
Priors are represented as callables::

    logprior(theta) -> float`
```
The formatting as-is is markdown. We're using rST.